apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -213,6 +218,16 @@ func verifyScaledObjects(incomingSo *ScaledObject, action string) error {
 		}
 	}
 
+	// verify ScalingModifiers structure if defined in ScaledObject
+	if incomingSo.Spec.Advanced != nil && !reflect.DeepEqual(incomingSo.Spec.Advanced.ScalingModifiers, ScalingModifiers{}) {
```

Comment:
As this check is used at least 3 times over the core, can we move it to the ScaledObject to reuse it? Something like, `.IsUsingModifiers()`

Reply:
I like this.